lockfile/plugin-commands-audit/src/audit.ts

```diff
@@ -105,6 +107,10 @@ export function help (): string {
             description: 'Use exit code 0 if the registry responds with an error. Useful when audit checks are used in CI. A build should fail because the registry has issues.',
             name: '--ignore-registry-errors',
           },
+          {
+            description: 'Ignore a vulnerability by CVE or GHSA. If no args are passed, ignore all CVEs with no resolution. Provide a comma separated list to ignore specific vulnerabilities. Specifying a GHSA ID will ignore all CVEs associated with the advisory.',
+            name: '--ignore-cve <cvelist>',
```

Comment:
if it also accepts GHSA, why is it named `--ignore-cve`?

Reply:
`--ignore` felt too generic, maybe `--ignore-vulnerabilities` makes more sense. Let me split out the PRs and can make that change in the new versions

Reply:
PR's have been split, assuming everything gets merged I will open up a 3rd PR that adds the functionality to pass a GHSA to the `--ignore-vulnerabilities` command